template/src/ontology/run.sh.jinja2

```diff
@@ -12,4 +12,14 @@
 
 IMAGE=${IMAGE:-odkfull}
 
-docker run -v $PWD/../../:/work -w /work/src/ontology {% if project.robot_java_args is defined %}-e ROBOT_JAVA_ARGS='{{ project.robot_java_args }}' -e JAVA_OPTS='{{ project.robot_java_args }}'{% endif %} {% if project.use_external_date is sameas True %}-e TODAY=`date +%Y-%m-%d` {% endif %}--rm -ti obolibrary/$IMAGE "$@"
+ODK_DEBUG=${ODK_DEBUG:-no}
+
+TIMECMD=
+if [ x$ODK_DEBUG = xyes ]; then
```

Comment:
I have never seen that pattern with the `x` - I can see its not wrong, but just out of curiosity - whats the reason?

Reply:
Suppose you have the following instead: `if [ $ODK_DEBUG = yes ]`. When `ODK_DEBUG` is not set (or is set to an empty string), that line will become `if [ = yes ]`, which is a syntax error (the shell would complain about a missing operand for the `=` operator or something like that). The `x` is here to avoid that.

An alternative is to quote the variable: `if [ "$ODK_DEBUG" = yes ]`, which would then result, if the variable is not defined, in `if [ "" = yes ]`. But some (mostly old) shells won’t accept that.

Reply:
Cheers, thanks for the info!